torch/_dynamo/eval_frame.py

```diff
@@ -360,9 +359,79 @@ def first_real_inst_idx(code):
     raise RuntimeError("RESUME instruction not found in code")
 
 
+# Converts a CodePart structure, which carries our
```

Comment:
"Convert" implies you are going to create a new plan. But you're not. You're mutably updating the frame state with new observations. This is another reason why "dynamic plan" was a bad name. It's mutably changing over time.

Reply:
Yea, I can change this